metricq_wizard_backend/metricq/configurator.py

```diff
@@ -406,6 +407,19 @@ async def create_client(self, *, token):
             config = await self.couchdb_db_config.create(token)
             await config.save()
 
+    async def delete_client(self, *, token):
```

Comment:
Would it be better to return a boolean here if *something* was deleted? Does it make sense that someone calls delete on a client that has neither a config nor a client entry - do we want this to cause an error?